src/init.cpp

```diff
@@ -1968,18 +1980,6 @@ bool AppInitMain(InitInterfaces& interfaces)
                 }
             }
 
-            std::vector<std::pair<std::string, uint16_t>> eth_endpoints;
-            std::vector<std::pair<std::string, uint16_t>> g_endpoints;
-            SetupRPCPorts(eth_endpoints, g_endpoints);
-
-            // Start the GRPC and ETH RPC servers
-            // Default to using the first address passed to bind
-            CrossBoundaryResult result;
-            start_servers(result, eth_endpoints[0].first + ":" + std::to_string(eth_endpoints[0].second), g_endpoints[0].first + "." + std::to_string(g_endpoints[0].second));
```

Comment:
This needs to be called before VerifyDB, moving it to after will cause VerifyDB to crash if it calls ConnectBlock on blocks with Eth TXs.

Reply:
Ah, great catch. Thanks! Will resolve.